src/test/java/de/tum/in/www1/artemis/authorization/AuthorizationTestService.java

```diff
@@ -13,17 +13,16 @@
 import org.springframework.web.method.HandlerMethod;
 import org.springframework.web.servlet.mvc.method.RequestMappingInfo;
 
-import de.tum.in.www1.artemis.security.annotations.EnforceAdmin;
-import de.tum.in.www1.artemis.security.annotations.EnforceNothing;
-import de.tum.in.www1.artemis.security.annotations.ManualConfig;
+import de.tum.in.www1.artemis.security.annotations.*;
 
 /**
  * This service is used to check if the authorization annotations are used correctly.
  */
 @Service
 public class AuthorizationTestService {
 
-    private static final Set<Class<? extends Annotation>> ALLOWED_AUTH_METHOD_ANNOTATIONS = Set.of(EnforceAdmin.class, EnforceNothing.class, PreAuthorize.class);
+    private static final Set<Class<? extends Annotation>> ALLOWED_AUTH_METHOD_ANNOTATIONS = Set.of(EnforceAdmin.class, EnforceAtLeastInstructor.class, EnforceAtLeastEditor.class,
+            EnforceAtLeastTutor.class, EnforceAtLeastStudent.class, EnforceNothing.class, PreAuthorize.class);
```

Comment:
Why is PreAuthorize still allowed?

Reply:
Mhh, I should rename it. It's allowed for the test to proceed.